src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java

```diff
@@ -7830,11 +7829,12 @@ public void testUpdateVenmoAccount_successful() throws Exception {
         assertThat(apiClientVenmoAccount, is(notNullValue()));
     }
 
+
     @Test
     public void testDocumentUpload_noUserToken() {
         Hyperwallet client = new Hyperwallet("test-username", "test-password");
         try {
-            client.documentUpload(null, new FormDataMultiPart());
+            client.documentUpload(null, null);
```

Comment:
I didn't find a better place to leave this comment...

@arao6 I would recommend to have two methods in the Hyperwallet.java:
- uploadUserDocuments >> **PUT** _/rest/v4/users/usr-123.../_
- uploadStakeholderDocuments >> **PUT** _/rest/v4/users/usr-123.../business-stakeholders/stk-222.../_

I was concerned about `documentUpload` method name.

Reply:
resolved